frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -86,6 +86,28 @@ describe("scenarios > question > new", () => {
   });
 
   describe("ask a (simple) question", () => {
+    it("should allow to search and select saved questions", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Simple question").click();
+      cy.findByPlaceholderText("Search for a table...").type("Ord");
+      cy.contains("Orders, Count").click();
+      cy.contains("18,760");
+    });
+
+    it("should allow to search and select tables", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Simple question").click();
+      cy.findByPlaceholderText("Search for a table...").type("Ord");
+      cy.contains("Orders")
+        .eq(0)
+        .click();
```

Comment:
I am always wary when we use `.eq(n)` or `.first()`/`.last()` in a test (unless it's completely irrelevant which member in the array are we picking - like in the case with dots/bars in visualizations). I've seen tests fail before simply because the order changes depending on some context.
Not saying this is going to happen here. This is more like a general note.

`cy.contains("Orders");` will match every single item in the dropdown menu.

Btw, why do we display "Orders" question 2 times? One under "Sample Dataset" and one under "Our Analytics"?
![image](https://user-images.githubusercontent.com/31325167/117036017-450f3b80-ad05-11eb-8f58-62650fd4ac53.png)

Reply:
I think one of them's the table and the other one's the card, so could be disambiguated by futzing w/ the icon

Reply:
Yeah @howonlee is right that one is a table in the db and one is a saved question. I think it'd help if we matched what we do in the main search where we disambiguate via the location description below the title.

Reply:
Oh, that makes sense.
But we can definitely help users make that distinction more easily. Can we also use different icons or is that too big hassle for now?

Reply:
Fixed that:
<img width="353" alt="Screen Shot 2021-05-05 at 02 16 13" src="https://user-images.githubusercontent.com/14301985/117081097-e0c19b80-ad47-11eb-936e-448563457af9.png">

Also, removed all `contains` and `eq`